sql/common/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/st_functions.scala

```diff
@@ -380,5 +380,9 @@ object st_functions extends DataFrameAPI {
 
   def ST_HausdorffDistance(g1: String, g2: String, densityFrac: Double) = wrapExpression[ST_HausdorffDistance](g1, g2, densityFrac);
 
+  def ST_CoordDim(geometry: Column): Column = wrapExpression[ST_CoordDim](geometry)
```

Comment:
You didn't implement test cases for testing this DataFrame style function.

Reply:
Could you please elaborate on where I must implement the test cases for this? I implemented it in functionTestScala.scala

Reply:
Please read [dataFrameAPITestScala.scala](https://github.com/apache/sedona/commit/b6702d1adacb98d83cc470e9b24877ab7785cdf8#diff-1346635eec60a5a0caf33a991414d06123a07299cb24d059af6424b88b1d3bfd)